sal_interfaces/ScriptQueue/ScriptQueue_Events.xml

```diff
@@ -61,7 +59,125 @@
     <Subsystem>ScriptQueue</Subsystem>
-    <Version>1.0.0</Version>
+    <Author></Author>
+    <EFDB_Topic>ScriptQueue_logevent_nextVisit</EFDB_Topic>
+    <Alias>nextVisit</Alias>
+    <Description>Group ID and other information about the next script to be run.</Description>
+    <item>
+        <EFDB_Name>salIndex</EFDB_Name>
+        <Description>Index of Script SAL component.</Description>
+        <IDL_Type>int</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>groupId</EFDB_Name>
+        <Description>Group ID for images taken by the script.</Description>
+        <IDL_Type>string</IDL_Type>
+        <IDL_Size>1</IDL_Size>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>coordinateSystem</EFDB_Name>
+        <Description>Coordinate system for position. A Script MetadataCoordSys enum. Observed is refracted apparent topocentric, e.g. similar to Mount but the telescope model is applied and the azimuth is wrapped. If None then position should be ignored.</Description>
+        <IDL_Type>short</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+        <Enumeration>CSys_None, CSys_ICRS, CSys_Observed, CSys_Mount</Enumeration>
+    </item>
+    <item>
+        <EFDB_Name>position</EFDB_Name>
+        <Description>Longitude, latitude axes of position in coordinateSystem</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>degree</Units>
+        <Count>2</Count>
+    </item>
+    <item>
+        <EFDB_Name>rotationSystem</EFDB_Name>
+        <Description>Rotation system for cameraAngle, a Script MetadataRotSys enum
+        * None: The camera rotator is not used; ignore cameraAngle.
+        * Sky: Position angle of focal plane +Y, measured from N through E.
+          At 0 degrees, +Y is along N and +X is along E/W depending on handedness.
+          At 90 degrees, +Y is along E and +X is along S/N depending on handedness.
+        * Horizon: Position angle of focal plane +Y, measured from +Alt through +Az.
+          At 0 degrees, +Y is along +Alt and +X is along +/-Az, depending on handedness.
+          At 90 degrees, +Y is along +Az and +X is along -/+Alt, depending on handedness.
+        * Mount: The position sent to the instrument rotator; the details depend on the rotator.
+        </Description>
+        <IDL_Type>short</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>cameraAngle</EFDB_Name>
+        <Description>Camera angle in rotationSystem</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>degree</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>filters</EFDB_Name>
+        <Description>Comma-separated names of acceptable filters, or blank for any filter.</Description>
+        <IDL_Type>string</IDL_Type>
+        <IDL_Size>1</IDL_Size>
+        <Units>second</Units>

```

Comment:
Unitless I think.

Reply:
Thank you for catching these errors in units. I also fixed the same errors in the Script metadata event.